### PR TITLE
Adding monitoring of an RDS instance

### PR DESCRIPTION
**Pull request by peterneubauer**

### CODE REVIEW

**Comment by aphyr** [comment]
Delightful. Thanks, @peterneubauer :)

File: bin/riemann-aws-rds-status [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+$0 = __FILE__ # Let's not expose our AWS keys in the process list
+
+class Riemann::Tools::AWS
+  include Riemann::Tools
+
+  opt :access_key, "AWS access key", :type => String
+  opt :secret_key, "Secret access key", :type => String
+  opt :region, "AWS region", :type => String, :default => 'eu-west-1'
+  opt :dbinstance_identifier, "DBInstanceIdentifier", :type => String
```
Comment (jamtur01) [inline bin/riemann-aws-rds-status]: Should there be a default here? Won't you get a .metric_name at https://github.com/aphyr/riemann-tools/pull/109/files#diff-91de02aa50aadb39fa72dbbc2344eae3R35 ?

Comment (jamtur01) [inline bin/riemann-aws-rds-status]: Or possible a fail if not specified? What's the behavior if you don't specify?

Comment (peterneubauer) [inline bin/riemann-aws-rds-status]: it should fail. You need ot know what RDS instance to run against. Didn't know how to do it :/

Comment (jamtur01) [inline bin/riemann-aws-rds-status]: Add something like this to initialize:

```ruby
abort "FATAL: specify a DB instance name, see --help for usage" unless opts[:dbinstance_identifier]
```

Comment (peterneubauer) [inline bin/riemann-aws-rds-status]: Thanks, PR coming!

**Pull request merged**